kiddraw.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
it's useful to use Rprojects as a way to organize code and get a good `.gitignore` file. they're helpful for keeping track of what you're doing.